<?xml version="1.0"?>
<launch>
    <node pkg="velodyne_pointcloud" type="cloud_node" name="convert_veloscans">
	<param name="calibration" value="$(find ndt_fuser)/launch/64E_calibration.yaml" />
    </node>
    <node pkg="ndt_fuser" type="ndt_fuser_node" name="fuser_3d" output="screen">
	<param name="points_topic" value="/velodyne_points" />
	<param name="matchLaser" value="false" />
	<param name="size_x_meters" value="110" />
	<param name="size_y_meters" value="110" />
	<param name="size_z_meters" value="80" />
	<param name="resolution" value="1" />
	<param name="sensor_range" value="110." />
	
	<!-- Specific sensor offset parameters for the example file -->
	<param name="sensor_pose_x" value="1.92" />
	<param name="sensor_pose_z" value="3.68" />
	<param name="sensor_pose_t" value="0.03" />
	<!--
	<param name="pose_init_x" value="10.73" />
	<param name="pose_init_y" value="2.185" />
	<param name="pose_init_t" value="0.05" />
	-->
	<param name="map_directory" value="$(find ndt_fuser)/maps/" />	
	<param name="map_name_prefix" value="kjula3d" />
	
	<param name="beHMT" value="true" />
	<param name="match2D" value="flase" />
	<param name="useOdometry" value="false" />
	<param name="visualize" value="true" />
	<param name="plotGTTrack" value="false" />
	<param name="initPoseFromGT" value="false" />
    </node>
</launch>
